src/test/java/jsUtils.js

```diff
@@ -1,12 +1,15 @@
+// providing a wait time which requires seconds as argument
 function waitTimeSec(x) {
     karate.log('sleeping');
     java.lang.Thread.sleep(x * 1000);
 }
 
+//generating random string which requires character length as argument
```

Comment:
What about the below format of the documentation?
```
/**
 * Generate random string
 * @param {count of the string} x 
 * @returns random string based upon the count provided 
 */
```

Reply:
ok will do it

Reply:
@yatheendra-egov sounds good. Please address one pending comments.

Reply:
done